tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1,7 +1,7 @@
 import { Download, Locator, Page, expect } from '@playwright/test'
 import util from 'util'
 import path from 'path'
-import { resourceExists, waitForResources, resourceNameSelector } from './utils'
+import { resourceExists, waitForResources } from './utils'
```

Comment:
this is required. just need to export the variable
```suggestion
import { resourceExists, waitForResources, resourceNameSelector } from './utils'
```